src/main/include/include.h

```diff
@@ -1,3 +1,6 @@
+
+//I coppy pasted this from 2020 code. I dont think I ever used it
+
```

Comment:
Ctrl+Shift+F in VS Code will tell you if `include.h` is ever referenced, and if it is not, we can delete this.